widgetssdk/src/main/java/com/glia/widgets/chat/adapter/holder/imageattachment/VisitorImageAttachmentViewHolder.java

```diff
@@ -34,6 +37,25 @@ public VisitorImageAttachmentViewHolder(
     public void bind(AttachmentFile attachmentFile, boolean showDelivered) {
         super.bind(attachmentFile);
         setShowDelivered(showDelivered);
+
+        if (showDelivered) {
+            itemView.setContentDescription(itemView.getResources().getString(R.string.glia_chat_visitor_image_delivered_content_description));
+        } else {
+            itemView.setContentDescription(itemView.getResources().getString(R.string.glia_chat_visitor_image_content_description));
+        }
+        ViewCompat.setAccessibilityDelegate(itemView, new AccessibilityDelegateCompat() {
```

Comment:
maybe can be moved to the base class `ImageAttachmentViewHolder` to avoid duplicate code in siblings?

Reply:
fixed